dev.vencord.Vesktop.yml

```diff
@@ -71,3 +72,41 @@ modules:
         x-checker-data:
           type: electron-updater
           url: https://github.com/Vencord/Vesktop/releases/latest/download/latest-linux-arm64.yml
+
+    modules:
+      # borrowed and tweaked from the net.lutris.Lutris manifest and the Arch PKGBUILD
+      - name: libspeechd
+        config-opts:
+          - --disable-static
+          - --with-ibmtts=no
+          - --with-kali=no
+          - --with-baratinoo=no
+          - --with-voxin=no
+          - --without-flite
+          - --disable-python
+          - --with-systemdsystemunitdir=/app/lib/systemd/system/speech-dispatcherd.service
+        no-make-install: true
+        post-install:
+          - cd ./src/api/c && make install
+        cleanup:
+          - "*.la"
+          - "*.a"
+          - /include
+        sources:
+          - type: archive
+            url: https://github.com/brailcom/speechd/releases/download/0.11.5/speech-dispatcher-0.11.5.tar.gz
+            sha512: d6d880bce0ae5bc2a5d519ef7740c689ae8b4b0bb658379762810e4beae3e465a429fbe19f7c490e89db0ea6a36aedd4b2287ac9251b90059b5c2cb3c0dd8a28
+        modules:
+          - name: dotconf
```

Comment:
what exactly is this needed for or used by?

Reply:
dotconf is a dependency of speech-dispatcher. It provides utility functions to parse config files. It's really only used in the server, which we aren't building, but it needs to exist to get past the ./configure step.

Reply:
would be good to add a comment stating this

i got a bit confused by it, especially because it's such an unpopular repo